rhc-ose-ansible/playbooks/registry/roles

```diff
@@ -0,0 +1 @@
+../../roles
```

Comment:
This allows the playbook to find the invoked roles. By convention, ansible will look in a folder called roles. It is being symbolically linked to the roles folder a few directories above